test-data/unit/check-classes.test

```diff
@@ -5180,8 +5180,8 @@ A = G
 x: A[B[int]]
 B = G
 [out]
-main:7: error: Type argument "builtins.int" of "G" must be a subtype of "builtins.str"
 main:7: error: Type argument "__main__.G[builtins.int]" of "G" must be a subtype of "builtins.str"
+main:7: error: Type argument "builtins.int" of "G" must be a subtype of "builtins.str"
```

Comment:
Maybe add `--show-column-numbers` in this test? (IIUC columns caused the change of order.)